src/app/muban/knockoutUtils.ts

```diff
@@ -10,7 +10,7 @@ import extract from 'html-extract-data';
  */
 export function initTextBinding(element: HTMLElement, html: boolean = false): Observable<string> {
   // init the observable with the correct initial data
-  const obs = ko.observable(<string>element[html ? 'innerHTML' : 'textContent']);
+  const obs = ko.observable(element[html ? 'innerHTML' : 'textContent'] as string);
```

Comment:
The returned value could be `null`, so `?? ''` is safer.

```suggestion
  const obs = ko.observable(element[html ? 'innerHTML' : 'textContent'] ?? '');
```